Models/categoriesModels.js

```diff
@@ -42,13 +42,22 @@ exports.updateReview = (review_id, inc_votes)=>{
         WHERE review_id = $1
         RETURNING *`,
         params
-    ).then((result) =>{
-        if (result.rows.length === 0) {
-            return Promise.reject({
-                status: 404,
-                msg: 'no review found with the input id, sorry!',
-            });
-        }
-        return result.rows[0]
-    })
-};
\ No newline at end of file
+        ).then((result) =>{
+            if (result.rows.length === 0) {
```

Comment:
if you like, instead of calculating the length you can use the rowsCount property that lives inside result already